spec/controllers/registrations_controller_spec.rb

```diff
@@ -50,7 +50,10 @@
     context "with invalid params" do
       context "empty fields" do
         before(:each) do
-          post :create, user: { name: "", email: "", password: "", password_confirmation: "", gender: "male" }
+          post :create,
+            params: {
+              user: { name: "", email: "", password: "", password_confirmation: "", gender: "male" }
+            }
         end
 
         it "shouldn't create the user" do
```

Comment:
Don't use should. Use for example: `'creates the user'`.
http://www.betterspecs.org/#should